span-normalizer/src/test/java/org/hypertrace/core/spannormalizer/fieldgenerators/HttpFieldsGeneratorTest.java

```diff
@@ -567,32 +567,54 @@ public void testResponseStatusCodeTagKeysPriority() {
   }
 
   @Test
-  public void testInvalidUrl() {
+  public void testRelativeUrl() {
```

Comment:
I'd suggest we describe the test case in here instead of describing what the input is. Something like `testRelativeURLExtractsPathAndQueryParams` tells me what this test is asserting, otherwise I have to read the code and guess the intention which makes it difficult the review.

Reply:
Makes sense. Made this change.